api/src/test/java/org/openmrs/OrderTest.java

```diff
@@ -632,6 +662,112 @@ public void isExpired_shouldReturnTrueIfDateStoppedIsNullAndAutoExpireDateIsBefo
 	}
 	
 	/**
+	 * @see Order#isActivated()
+	 * @verifies return true if an order was activated on the current date
+	 */
+	@Test
+	public void isActivated_shouldReturnTrueIfAnOrderWasActivatedOnTheCurrentDate() throws Exception {
+		Order order = new Order();
+		order.setDateActivated(new Date());
```

Comment:
In theory, you should not duplicate these tests for both isActivated() and isActivated(checkDate), all tests should be against isActivated(checkDate) except for one where you test that when isActivated() is called it defaults to now